src/typings/idevice-app-launcher.d.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+/// <reference path="./node.d.ts" />
+/// <reference path="./q.d.ts" />
+
+declare module "idevice-app-launcher" {
```

Comment:
How about including this typing file in https://github.com/Microsoft/idevice-app-launcher ?